### PR TITLE
WIP: Cleanup warnings

### PR DESCRIPTION
**Pull request by ombharatiya**
## Context

UI tests have lots of ember & sinon.js warnings on deprecated function usage

## Objective

Cleanup Deprecated warnings

## References

https://github.com/screwdriver-cd/screwdriver/issues/2493

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: app/pipeline/events/controller.js [hunk 1]
```diff
@@ -390,6 +393,11 @@ export default Controller.extend(ModelReloaderMixin, {
   selectedEvent: computed('selected', 'mostRecent', {
```
Comment (adong) [inline app/pipeline/events/controller.js:393]: You could just update to 

```
selectedEvent: computed('selected', 'mostRecent', function selectedEvent() {
	// return 
    // logics
});
```

for getters only, `set` is only necessary if we actually _used_ `set`. 

This way is much easier to refactor

Comment (ombharatiya) [reply app/pipeline/events/controller.js:393]: We're using `set` in test files and that's why it's useful.

Comment (adong) [reply app/pipeline/events/controller.js:393]: Just want to point out that: `this.set('selectedEvent', someVar)` ≠ [`this.selectedEvent = someVar`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/set)

Ref: https://stackoverflow.com/a/55937966

**Comment by tkyi** [comment]
Closing as this PR is 3 months old
Feel free to reopen if you're still actively working on it

**Pull request closed without merging**